src/EnergyPlus/OutputReportTabularAnnual.cc

```diff
@@ -246,6 +249,65 @@ namespace EnergyPlus {
 			}
 		}
 
+		void
+		checkAggregationOrderForAnnual( )
+		{
+			std::vector<AnnualTable>::iterator annualTableIt;
+			bool invalidAggregationOrderFound = false;
+			if ( !DataGlobals::DoWeathSim ) {// if no weather simulation than no reading of MonthlyInput array
+				return;
+			}
+			for ( annualTableIt = annualTables.begin( ); annualTableIt != annualTables.end( ); ++annualTableIt ) {
+				if ( annualTableIt->invalidAggregationOrder( ) ) {
+					invalidAggregationOrderFound = true;
+				}
+			}
+			if ( invalidAggregationOrderFound ) {
+				ShowFatalError( "OutputReportTabularAnnual: Invalid aggregations detected, no simulation performed." );
+			}
+		}
```

Comment:
Just so all instances of this issues are reported in the error file prior to the fatal.